Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment-Legacy.ps1

```diff
@@ -0,0 +1,134 @@
+param
```

Comment:
@ericsciple  What is correct solution to this problem ?
- Should I create legacy class for Utility and AzureUtility too